### PR TITLE
Caryn Keffer

### PR DESCRIPTION
**Pull request by carynkeffer**

### CODE REVIEW

File: lib/school.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    @student_names       = []
+  end
+
+  def add_student_name(name)
+    @student_names << name
+  end
+
+  def military_time
+    start_time[0].to_i + hours_in_school_day
+  end
```
Comment (memcmahon) [inline lib/school.rb:17]: Nice helper method - and great job testing it!

File: lib/school.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    military_time.to_s + ":00"
+  end
+
+  def is_full_time?
+    if @hours_in_school_day >= 4
+      true
+    else
+      false
+    end
+  end
```
Comment (memcmahon) [inline lib/school.rb:29]: You could also rely on the return value of the conditional statement here:

```ruby
def is_full_time?
  @hours_in_school_day >= 4
end
```

File: lib/school.rb [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  def ante_meridiem
+    if military_time >= 12
+      convert_end_time_to_clock_time + " PM"
+    else
+      convert_end_time_to_clock_time + " AM"
+    end
+  end
+
+  def convert_end_time_to_clock_time
+    if military_time > 12
```
Comment (memcmahon) [inline lib/school.rb:54]: Your test for this is passing, but by coincidence.  I think you actually want to be calling `clock_time` instead here.  See what happens if you have a school that looks like this: `@school2 = School.new('10:00', 2)`

**Review by memcmahon — COMMENTED** [review]
🎉 Great Job - theres just a bit of weirdness in that last method; by coincidence, the error in your method is not being caught by your test; but you are on the right track, for sure.

Score: 3.5

File: lib/school.rb (hunk 3 above)
Comment (carynkeffer) [reply lib/school.rb:54]: Oh... yes I see... ! Thank you!

File: lib/school.rb (hunk 2 above)
Comment (carynkeffer) [reply lib/school.rb:29]: @memcmahon Thank you!

**Pull request closed without merging**